tbd.schema.json

```diff
@@ -441,6 +469,106 @@
           "title": "Global state",
           "type": "string"
         },
+        "ua": {
+          "title": "UA' assessment",
+          "type": "object",
+          "properties": {
+            "bloc1": {
+              "title": "heating setpoints >= 18°C",
+              "type": "object",
+              "properties": {
+                "summary": {
+                  "type": "string"
+                },
+                "walls": {
+                  "type": "string"
+                },
+                "roofs": {
+                  "type": "string"
+                },
+                "floors": {
+                  "type": "string"
+                },
+                "doors": {
+                  "type": "string"
+                },
+                "windows": {
+                  "type": "string"
+                },
+                "skylights": {
+                  "type": "string"
+                },
+                "rimjoists": {
+                  "type": "string"
+                },
+                "parapets": {
+                  "type": "string"
+                },
+                "trim": {
+                  "type": "string"
+                },
+                "corners": {
+                  "type": "string"
+                },
+                "grade": {
+                  "type": "string"
+                },
+                "other": {
+                  "type": "string"
+                }
+              },
+              "additionalProperties": false,
+              "required": ["summary"]
+            },
+            "bloc2": {
+              "title": "heating setpoints >= 18°C",
```

Comment:
Typo? Should it be "bloc2: spaces/zones with HSTP < 18°C"?

Reply:
That's a lazy copy/paste-driven error. Will fix.